README.md

```diff
@@ -189,6 +191,23 @@ createLwsIotCredentialProvider(
 freeIotCredentialProvider(&pSampleConfiguration->pCredentialProvider);
 ```
 
+## Use Pre-generated Certificates
+The certificate generating function (createCertificateAndKey) in createDtlsSession() can take between 5 - 15 seconds in low performance embedded devices, it is called for every peer connection creation when KVS WebRTC receives an offer. To avoid this extra start-up latency, certificate can be pre-generated and passed in when offer comes.
```

Comment:
Include the RFC link - might be useful. https://www.w3.org/TR/webrtc/#dom-rtcconfiguration-certificates